tools/mod-builder/common.py

```diff
@@ -67,13 +74,17 @@ def request_user_input(first_option: int, last_option: int, error_msg: str) -> i
     i = 0
     while True:
         try:
-            i = int(input())
+            i = int(input()) if len(sys.argv) < 2 or len(remaining_args) == 0 else int(remaining_args.pop(0))

```

Comment:
Assuming the user is using the cli args, the `input()` will only trigger if you've consumed all the tokens already (i.e, your last command passed via command line is incomplete). Then, after the user gives a valid `input()`, you're terminating the program. If the intention is to use the cli args without a fallback to manual inputting, then I'd suggest making this comparison outside of the `try/catch` and raising an exception if the cli args are incomplete, so there's a feedback to the user on what went wrong logged in `crash.log`.